src/signature_pad.js

```diff
@@ -92,11 +92,12 @@ SignaturePad.prototype.clear = function () {
   this._isEmpty = true;
 };
 
-SignaturePad.prototype.fromDataURL = function (dataUrl) {
+SignaturePad.prototype.fromDataURL = function (dataUrl, options) {
```

Comment:
Indeed. I wasn't sure so I followed the pattern of other functions.

I changed it now, this is what it compiles to:

```js
SignaturePad.prototype.fromDataURL = function (dataUrl) {
  var _this = this;

  var options = arguments.length > 1 && arguments[1] !== undefined ? arguments[1] : {};

  var image = new Image();
  var ratio = options.ratio || window.devicePixelRatio || 1;
  var width = options.width || this._canvas.width / ratio;
  var height = options.height || this._canvas.height / ratio;
  // ...

```